generator/templates/enums/template.m

```diff
@@ -0,0 +1,11 @@
+{% include 'copyright.txt' %}
+
+#import "{{name}}.h"
+{%- block body %}
+{% if add_typedef %}
+typedef SDLEnum {{name}} SDL_SWIFT_ENUM;
+{% endif -%}
+{%- for param in params %}
+{{ name }} const {{ name }}{{param.name}} = @"{{param.origin}}";
```

Comment:
Strings that match a keyword should never be changed.

Reply:
`@"{{param.origin}}` will never be changed.
`{{param.name}}` could be changed if much with with following logic:
```python3
if any(re.search(r'^(sdl)?({})?{}$'.format(item_name.casefold(), name.casefold()), k) for k in self.key_words):
```
done in https://github.com/smartdevicelink/sdl_ios/pull/1556/commits/8d538469711fe0690157629c5f8d0c361b29cdff